tests/junit-xml/.travis.yml

```diff
@@ -1,18 +1,38 @@
 language: python
 
+matrix:
+  include:
+    - python: pypy
+      dist: xenial
+      sudo: true
+      env: TOXENV=pypy
+    - python: 3.5
+      dist: xenial
+      sudo: true
+      env: TOXENV=py35
+    - python: 3.7
+      dist: xenial
+      sudo: true
+      env: TOXENV=py37
+    - python: 3.8
+      dist: xenial
+      sudo: true
+      env: TOXENV=py38
+
 env:
-    - TOXENV=py26
-    - TOXENV=py27
-    - TOXENV=pypy
-    - TOXENV=py33
-    - TOXENV=py34
-    - TOXENV=cover
+  - TOXENV=py27
+  - TOXENV=py36
+  - TOXENV=cover
+  - TOXENV=flake8
 
 install:
-    - travis_retry pip install tox==1.6.1
+  - travis_retry pip install tox==3.13.2
 
 script:
-    - travis_retry tox
+  - travis_retry tox
+
+git:
+  depth: 1
 
 notifications:
   email:
```

Comment:
We don't need or use this file. No idea if upstream does either.

Reply:
I deleted `travis.yml`